test/feed-entrypoint.test.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const feedEntrypoint = require('../lib/feed-entrypoint');
+
+test('determines entrypoint for simple feed', async () => {
+    const feed = [{ id: 1, file: '', source: '', deps: {}, entry: true }];
+
+    expect(feedEntrypoint(feed).id).toBe(1);
+});
+
+test('determines entrypoint for feed', async () => {
+    const feed = [
+        { id: 1, file: '', source: '', deps: {} },
+        { id: 2, file: '', source: '', deps: {} },
+        { id: 3, file: '', source: '', deps: {}, entry: true },
+        { id: 4, file: '', source: '', deps: {} },
+    ];
+
+    expect(feedEntrypoint(feed).id).toBe(3);
+});
+
+test('determines entrypoint for feed with multiple entrypoints', async () => {
```

Comment:
does this test make any sense? When do you have multiple entrypoints in a single feed?

Reply:
I don't think you can so yea, could probably remove.